bib/cmd/bootc-image-builder/main.go

```diff
@@ -40,10 +39,7 @@ func check(err error) {
 }
 
 type BuildConfig struct {
```

Comment:
If the build config is going to only be a blueprint then making it a top level object and accepting TOML would make sense to me ;)

Reply:
Yeah, that's something I would like to look into as the next step. This is a left-over from when we hacked up this tool during one afternoon from random bits in `osbuild/images`. :sweat_smile: